airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/run.py

```diff
@@ -4,15 +4,52 @@
 
 
 import sys
+import traceback
+from datetime import datetime
+from typing import List
 
-from airbyte_cdk.entrypoint import launch
+from orjson import orjson
+
+from airbyte_cdk.entrypoint import AirbyteEntrypoint, launch, logger
+from airbyte_cdk.exception_handler import init_uncaught_exception_handler
+from airbyte_cdk.models import AirbyteErrorTraceMessage, AirbyteMessage, AirbyteMessageSerializer, AirbyteTraceMessage, TraceType, Type
 from source_google_analytics_data_api import SourceGoogleAnalyticsDataApi
-from source_google_analytics_data_api.config_migrations import MigrateCustomReports, MigrateCustomReportsCohortSpec, MigratePropertyID
 
 
-def run():
-    source = SourceGoogleAnalyticsDataApi()
-    MigratePropertyID.migrate(sys.argv[1:], source)
-    MigrateCustomReports.migrate(sys.argv[1:], source)
-    MigrateCustomReportsCohortSpec.migrate(sys.argv[1:], source)
```

Comment:
We should retain these migrations. Was there a particular reason why we have removed them? They can still be used in `run()` when we create the low-code source

Reply:
I believe we don't need this migration, as we implemented it some time ago, and I think all configurations have already been moved to the new version of the spec.